.github/workflows/build-tests.yml

```diff
@@ -1,107 +0,0 @@
-name: build-tests
```

Comment:
what happened to these?

Reply:
moved to tests.yml!

- the events were essentially the same anyway
- I got to rename these jobs to match the same naming scheme as tests.yml
- the workflow diagram is also easier to understand now that all jobs are on the same page
- this will let us consolidate the dependencies a bit nicer going forward